_dev/src/App.vue

```diff
@@ -56,7 +56,37 @@
         <p>{{ $t('general.testModeOn') }}</p>
       </b-alert>
     </div>
-    <router-view />
+
+    <div
+      class="container"
+      v-if="!isShopContext"
+    >
+      <b-alert
+        variant="warning"
+        show
+      >
+        <h2>{{ $t('general.multiShop.title') }}</h2>
+        <p>{{ $t('general.multiShop.subtitle') }}</p>
+        <p>{{ $t('general.multiShop.chooseOne') }}</p>
+        <b-list-group
+          v-for="group in shopsTree"
+          :key="group.id"
+          class="mt-3 mb-3 col-4"
+        >
+          <p class="text-muted">{{ $t('general.multiShop.group') }} {{ group.name }}</p>
+          <b-list-group-item
+            v-for="shop in group.shops"
+            :key="shop.id"
+            :href="shop.url"
+          >
+            {{ shop.name }}
```

Comment:
If you plan to display buttons, do it the same way as the message above it with `<b-button>` so everything remain coherent:

![image](https://user-images.githubusercontent.com/6768917/75180109-6a80dc80-5733-11ea-87fd-8aa00031ae7e.png)

Having a call to action would help too, something like "Configure xxx" or "Switch on xxx"

Reply:
Based on screen made by UX
It's not buttons but list elements, idea is to have a list of shops displayed by group like the native shop selector in header

Reply:
Following Nadia's feedback, roundings may be not be applied to the proper context.

Shall we display the alerts about rounding & the Sandbox module only whenthe context is on a specific shop?

Reply:
Following Nadia's feedback, roundings may be not be applied to the proper context.

Shall we display the alerts about rounding & the Sandbox module only when the context is on a specific shop?

Reply:
Oh right, as long as you can't display PrestaShop Checkout in MultiShop, make no sense to display roudings banner or sandbox mode banner in multishop neither.

For the call to action, not a bad idea.
Could we display 
"Group: Default
Configure PrestaShop
Configure Doge
Configure Default Shop"

?

Reply:
Oh right, as long as you can't display PrestaShop Checkout in MultiShop, make no sense to display roudings banner or sandbox mode banner in multishop neither.

For the call to action, not a bad idea.
Could we display 
"Group: Default
Configure PrestaShop
Configure Doge
Configure Default Shop"

?

Reply:
I added Configure before shop name